src/api/users/test/e2e/users.test.js

```diff
@@ -367,15 +380,18 @@ describe('DELETE REQUESTS', () => {
 
     const response = await request(app)
       .delete(`/${user.id}`)
+      .set('Authorization', `bearer ${createServiceToken()}`)
       .set('Content-Type', 'application/json');
 
     expect(response.statusCode).toBe(200);
   });
 
   test('Rejected - Deleted a nonexistent user', async () => {
     const id = createUserHash('no-such-user@email.com');
-    const response = await request(app).delete(`/${id}`).set('Content-Type', 'application/json');
-
+    const response = await request(app)
+      .delete(`/${id}`)
+      .set('Content-Type', 'application/json')
+      .set('Authorization', `bearer ${createServiceToken()}`);
     expect(response.statusCode).toBe(404);
```

Comment:
Do we want to send a 404 status code here? Wouldn't we want to send a 303 and redirect so that when a user is deleted, they go back to the main telescope page instead?

Reply:
The Users service has no concept of Telescope.  It's just processing a request to delete a user that isn't there, hence 404.  With microservices, you need to constrain your view to the request/response paradigm vs. thinking in terms of the larger app.  We might choose to do some redirect in the front-end based on this 404 later on, but that's a different issue.